config/bootstrap.php

```diff
@@ -43,27 +42,6 @@
     ]);
 }
 
-Router::plugin('DebugKit', function ($routes) {
```

Comment:
Not sure what you mean. `Plugin::routes('DebugKit')` won't even try loading routes if the plugin isn't loaded with `'routes' => true`. (see: https://github.com/cakephp/cakephp/blob/master/src/Core/Plugin.php#L335)